src/styles/card.scss

```diff
@@ -0,0 +1,61 @@
+.card {
+  @include space-between;
+  flex-direction: column;
+  width: 200px;
+  margin-bottom: 50px;
+  padding: 30px 14px 15px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 35px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 170px;
+  }
+
+  &__title {
+    margin-bottom: 5px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 15px;
+  }
+
+  &__code {
+    margin-bottom: 15px;
+  }
+
+  &__code,
+  &__review,
+  &__price-label {
+    font-weight: 400;
+  }
```

Comment:
Do not combine selectors with `,` operator, it's hard to find them later.
Use mixin.

Reply:
fixed

Reply:
fixed